### DISCUSSION TITLE
Add custom HTTP header

### DISCUSSION THREAD

**Discussion by sberer** [regulaforensics/DocumentReader-web-csharp-client#28] [Q&A]
We use an OnPremise installation of the Regula Web SDK. The Regula Web SDK runs on a server which is located behind a gateway. To authenticate against the gateway, it is necessary to send a custom HTTP header (i.e. Subscription-Key: 1234-ABC). Only if the authentication at the gateway is successful, the request is forwarded to the Regula server. Is it currently possible to set such a HTTP header via the SDK? Can you provide an example?

**Comment by hleb-albau** [comment]
Checkout 5.7.3 version, we add the ability to pass custom HTTP headers for each request!

**Comment by sberer** [comment]
I just upgraded to the latest SDK version. Why was this functionality removed? I need to pass custom HTTP headers OR custom query parameters to authenticate against our gateway.
